src/box/lua/net_box.c

```diff
@@ -2313,6 +2394,62 @@ netbox_transport_on_event(struct netbox_transport *transport,
 	lua_call(L, watch.data != NULL ? 3 : 2, 0);
 }
 
+/**
+ * Write response to buffer.
```

Comment:
Nit: This comment is useless, because it just reiterates the function name. Please say what it does in more details, what it returns, and what happens on error.

Reply:
Done